tensorflow_addons/utils/resource_loader.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utilities similar to tf.python.platform.resource_loader."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+
+def get_project_root():
+    """Returns project root folder."""
+    return os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Could you create an unit test ? Say, assert `get_path_to_datafile('tf_dependency')` directory exists.

Reply:
I would really like a test here because this is specific to the file staying where it exists today.  The problem is I think the unittest will have issue verifying a directory

```
import os
import tensorflow as tf
from tensorflow_addons.utils.resource_loader import get_path_to_datafile


class ResourceLoaderTest(tf.test.TestCase):
    def test_get_project_root(self):
        dependancy_dir = get_path_to_datafile('tf_dependency')
        self.assertTrue(os.path.exists(dependancy_dir))


if __name__ == '__main__':
    tf.test.main()
```
Fails.

And if I force a comparison of the string `dependancy_dir` I see:
```
AssertionError: '/root/.cache/bazel/_bazel_root/e051f2f195071208ea7f081f88730f4f/execroot/__main__/bazel-out/k8-fastbuild/bin/tensorflow_addons/utils/resource_loader_test.runfiles/__main__/tensorflow_addons/tf_dependency' != 'test'
```

Reply:
Oh, I get it. Does it work to check in runtime, I mean, in the `get_project_root` function?

Reply:
Hmmm at runtime 'tf_dependecy' won't exist in the package as the pip build is just `tensorflow_addons` and what's below it.  That reminded me that we should have been looking for say `examples` dir, but that also failed in the unit test. 

If `get_path_to_datafile` fails at runtime it'll throw a file not found error on the op load... so while bad it'll be an understandable error and obvious break. Open to adding a test but can't think of one that helps much.